tests/foreman/installer/test_new_installer.py

```diff
@@ -0,0 +1,122 @@
+"""Smoke tests to check installation health
+
+:Requirement: Installation
+
+:CaseAutomation: Automated
+
+:CaseComponent: Installation
+
+:Team: Rocket
+
+:CaseImportance: Critical
+
+"""
+
+from broker import Broker
+import pytest
+
+from robottelo.config import settings
+from robottelo.hosts import Satellite, get_sat_rhel_version
+
+pytestmark = [pytest.mark.foremanctl, pytest.mark.build_sanity, pytest.mark.upgrade]
+
+SATELLITE_SERVICES = [
+    'candlepin',
+    'dynflow-sidekiq@orchestrator',
+    'dynflow-sidekiq@worker',
+    'dynflow-sidekiq@worker-hosts-queue',
+    'foreman-proxy',
+    'foreman',
+    'httpd',
+    'postgresql',
+    'pulp-api',
+    'pulp-content',
+    'pulp-worker@*',
+    'redis',
+]
+
+
+def common_sat_install_assertions(satellite):
+    # no errors/failures in journald
+    result = satellite.execute(
+        r'journalctl --quiet --no-pager --boot --priority err -u "dynflow-sidekiq*" -u "foreman-proxy" -u "foreman" -u "httpd" -u "postgresql" -u "pulp-api" -u "pulp-content" -u "pulp-worker*" -u "redis" -u "candlepin"'
+    )
+    assert not result.stdout
+    # no errors/failures in /var/log/httpd/*
+    result = satellite.execute(r'grep -iR "error" /var/log/httpd/*')
+    assert not result.stdout
+    # # no errors/failures in /var/log/candlepin/*
+    result = satellite.execute(r'grep -iR "error" /var/log/candlepin/*')
+    assert not result.stdout
+    httpd_log = satellite.execute('journalctl --unit=httpd')
+    assert 'WARNING' not in httpd_log.stdout
+
+
+@pytest.fixture(scope='module')
+def module_sat_ready_rhel(request):
+    with Broker(
+        workflow=settings.server.deploy_workflows.os,
+        deploy_rhel_version=get_sat_rhel_version().major,
```

Comment:
Could we rely directly on settings here?
```suggestion
        deploy_rhel_version=settings.server.version.release,
```